index.Rmd

```diff
@@ -103,7 +103,7 @@ map_3 <- tm_shape(background_bu, bbox = boundary_di_bu) +
   tm_lines(scale = 4, col = "red") +
   tm_credits(paste("Service areas with line-based network 50m-buffer method is",as.character(round(area_RD/1000000)),"km2"), position=c("left", "bottom"))
   
-map_3
+map_3. 
 ```
 
-This image proposed a line-based network buffer method that defined areas near the center line of routes as accessible, which is more accurate, as it is closer to the actual environment available to pedestrians. As you can see in Map2 and Map3, the service areas determined by the route-based method were significantly less than those by Euclidean distance buffer method, which the ratio is `r as.numeric(round(round(area_RD/1000000)/round(area_ED/1000000), 2))`.
\ No newline at end of file
+This image proposed a line-based network buffer method that defined areas near the center line of routes as accessible, which is more accurate, as it is closer to the actual environment available to pedestrians. As you can see in Map2 and Map3, the service areas determined by the route-based method were significantly less than those by Euclidean distance buffer method, which the ratio is `r as.numeric(round(round(area_RD/1000000)/round(area_ED/1000000), 2))`.  
```

Comment:
Really interesting finding! I'd love to hear more of what you have to say in an expanded findings and conclusion section. The visuals are so interesting I want to know more about your analysis :)